pkg/reconciler/kubernetes/tektonresult/controller.go

```diff
@@ -59,21 +59,34 @@ func NewExtendedController(generator common.ExtensionGenerator) injection.Contro
 			logger.Fatalw("Error creating initial manifest", zap.Error(err))
 		}
```

Comment:
is it possible to use `InitController` here?

Reply:
as of now results doesn't have a version configmap defined, will be raising a PR on tektoncd/results and then we can use the InitController

Reply:
will wait till https://github.com/tektoncd/results/pull/167 merges and a new release of tektoncd/results happens. Can raise a follow-up PR later on

Reply:
will wait till https://github.com/tektoncd/results/pull/167 merges and a new release of tektoncd/results happens. Can raise a follow-up PR later on